src/components/common/Footer/index.js

```diff
@@ -4,20 +4,107 @@ import Link from 'gatsby-link';
 import withStyles from 'react-jss';
 import cx from 'classnames';
 
+import navigation from '../../../data/navigation.json';
 import logoReversed from '../../../assets/images/rwr/logo-mini-reversed.png';
+import youtube from '../../../assets/images/footer/youtube.png';
+import facebook from '../../../assets/images/footer/facebook.png';
+import twitter from '../../../assets/images/footer/twitter.png';
+import meetup from '../../../assets/images/footer/meetup.png';
 import styles from './styles';
+import { SignUp, Contact } from '../../../containers';
```

Comment:
A component can't import a container. The SignUp and Contact should come from their parents. Even if we have to import them in every page. However we are not building an app, at least we can decouple "a little" the business logic which we will add to some components and static components.